## 07_typos_fix.patch by  <bbyever@gmail.com>
##
##  fixes some typos in man pages.

diff -urNad ubuntu~/man/cryptsetup.8 ubuntu/man/cryptsetup.8
--- ubuntu~/man/cryptsetup.8	2008-05-25 20:27:00.000000000 +0200
+++ ubuntu/man/cryptsetup.8	2008-05-25 20:30:42.782915083 +0200
@@ -1,4 +1,4 @@
-.TH CRYPTSETUP "8" "March 2005" "cryptsetup 1.0.3" "Maintainance Commands"
+.TH CRYPTSETUP "8" "March 2005" "cryptsetup 1.0.3" "Maintenance Commands"
 .SH NAME
 cryptsetup - setup cryptographic volumes for dm-crypt (including LUKS extension)
 .SH SYNOPSIS
@@ -96,28 +96,28 @@
 set key size in bits. Has to be a multiple of 8 bits. The key size is limited by the used cipher. See output of /proc/crypto for more information. Can be used for \fIcreate\fR or \fIluksFormat\fR, all other LUKS actions will ignore this flag, as the key-size is specified by the partition header. Default is 128.
 .TP
 .B "\-\-size, \-b"
-force the size of the underlaying device in sectors.
+force the size of the underlying device in sectors.
 .TP
 .B "\-\-offset, \-o"
-start offset in the backend device.
+start offset in the back-end device.
 .TP
 .B "\-\-skip, \-p"
-how many sectors of the encrypted data to skip at the beginning. This is different from the \-\-offset options with respect to IV calculations. Using \-\-offset will shift the IV calculcation by the same negative amount. Hence, if \-\-offset \fIn\fR, sector \fIn\fR will be the first sector on the mapping with IV \fI0\fR. Using \-\-skip would have resulted in sector \fIn\fR being the first sector also, but with IV \fIn\fR.
+how many sectors of the encrypted data to skip at the beginning. This is different from the \-\-offset options with respect to IV calculations. Using \-\-offset will shift the IV calculation by the same negative amount. Hence, if \-\-offset \fIn\fR, sector \fIn\fR will be the first sector on the mapping with IV \fI0\fR. Using \-\-skip would have resulted in sector \fIn\fR being the first sector also, but with IV \fIn\fR.
 .TP
 .B "\-\-readonly"
 setup a read-only mapping.
 .TP
 .B "\-\-iter-time, \-i"
-The number of milliseconds to spend with PBKDF2 password processing. This options is only relevant to LUKS key setting operations as \fIluksFormat\fR or \fIluksAddKey\fR.
+The number of milliseconds to spend with PBKDF2 password processing. This option is only relevant to LUKS key setting operations as \fIluksFormat\fR or \fIluksAddKey\fR.
 .TP
 .B "\-\-batch-mode, \-q"
 Do not ask for confirmation. This option is only relevant for \fIluksFormat\fR.
 .TP
 .B "\-\-timeout, \-t"
-The number of seconds to wait before timeout. This option is relevant evertime a password is asked, like \fIcreate\fR, \fIluksOpen\fR, \fIluksFormat\fR or \fIluksAddKey\fR. It has no effect if used in conjunction with \-\-key-file.
+The number of seconds to wait before timeout. This option is relevant every time a password is asked, like \fIcreate\fR, \fIluksOpen\fR, \fIluksFormat\fR or \fIluksAddKey\fR. It has no effect if used in conjunction with \-\-key-file.
 .TP
 .B "\-\-tries, \-T"
-How often the input of the passphrase shall be retried. This option is relevant evertime a password is asked, like \fIcreate\fR, \fIluksOpen\fR, \fIluksFormat\fR or \fIluksAddKey\fR. The default is 3 tries.
+How often the input of the passphrase shall be retried. This option is relevant every time a password is asked, like \fIcreate\fR, \fIluksOpen\fR, \fIluksFormat\fR or \fIluksAddKey\fR. The default is 3 tries.
 .TP
 .B "\-\-align-payload=\fIvalue\fR"
 Align payload at a boundary of \fIvalue\fR 512-byte sectors.  This option is relevant for \fIluksFormat\fR.  If your block device lives on a RAID it is
@@ -134,7 +134,7 @@
 
 \fIFrom a key file\fR: It will be cropped to the size given by \-s. If there is insufficient key material in the key file, cryptsetup will quit with an error.
 
-If \-\-key-file=- is used for reading the key from stdin, no trailing newline ist stripped from the input. Without that option, cryptsetup strips trailing newlines from stdin input.
+If \-\-key-file=- is used for reading the key from stdin, no trailing newline is stripped from the input. Without that option, cryptsetup strips trailing newlines from stdin input.
 .SH NOTES ON PASSWORD PROCESSING FOR LUKS
 LUKS uses PBKDF2 to protect against dictionary attacks (see RFC 2898). 
 LUKS will always use SHA1 in HMAC mode, and no other mode is supported at the moment. 
@@ -147,12 +147,12 @@
 For any password creation action (luksAddKey, or luksFormat), the user specify, how much the time the password processing should consume. 
 Increasing the time will lead to a more secure password, but also will take luksOpen longer to complete. The default setting of one second is sufficient for good security.
 .SH NOTES ON PASSWORDS
-Mathematic can't be bribed. Make sure you keep your passwords safe. There are a few nice tricks for constructing a fallback, when suddely out of (or after being) blue, your brain refuses to cooperate. These fallbacks are possible with LUKS, as it's only possible with LUKS to have multiple passwords.
+Mathematics can't be bribed. Make sure you keep your passwords safe. There are a few nice tricks for constructing a fallback, when suddnely out of (or after being) blue, your brain refuses to cooperate. These fallbacks are possible with LUKS, as it's only possible with LUKS to have multiple passwords.
 .SH AUTHORS
 cryptsetup is written by Christophe Saout <christophe@saout.de>
 .br
 LUKS extensions, and man page by Clemens Fruhwirth <clemens@endorphin.org>
-.SH "COMPATABILITY WITH OLD SUSE TWOFISH PARTITIONS"
+.SH "COMPATIBILITY WITH OLD SUSE TWOFISH PARTITIONS"
 To read images created with SuSE Linux 9.2's loop_fish2 use \-\-cipher
 twofish-cbc-null \-s 256 \-h sha512, for images created with even
 older SuSE Linux use \-\-cipher twofish-cbc-null \-s 192 \-h
